include/deal.II/base/table.h

```diff
@@ -1903,7 +1903,7 @@ TableBase<N,T>::reinit (const TableIndices<N> &new_sizes,
   // if new size is nonzero: if necessary allocate additional memory, and if
   // not fast resize, zero out all values)
   values.resize_fast (new_size);
-  if (!fast)
+  if (!omit_zeroing_entries)
```

Comment:
`omit_zeroing_entries` might be a misnomer here: will the default constructor for everything we can put in a `Table` set its contents equal to zero?